Alcatraz/Categories/NSImage+Alcatraz.h

```diff
@@ -0,0 +1,13 @@
+//
+//  NSImage+Alcatraz.h
+//  Alcatraz
+//
+//  Created by Roman Gardukevich on 24/8/16.
+//  Copyright © 2016 supermar.in. All rights reserved.
+//
+
+#import <Cocoa/Cocoa.h>
+
+@interface NSImage (Alcatraz)
+@property (nonatomic, readonly) NSInteger numberOfFrames;
```

Comment:
I'd rename the method `-frameCount`. This is more common in Cocoa (eg. NSArray's `-count`) and matches with the property name `NSImageFrameCount`.